### PR TITLE
improving wording on index.md

### PR DESCRIPTION
**Pull request by albi3ro**
I went over the index page and improved flow.

I've found that I really like Plots.jl as a package, but I'd like to help the documentation. 

I haven't worked on someone else's project before, so I hope I'm doing this correctly.

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
+Data visualization has a complicated history.  Plotting software makes trade-offs between features and simplicity, speed and beauty, and a static and dynamic interface. Some packages make a display and never change it, while others make updates in real-time.
 
-Plots is a visualization interface and toolset.  It sits above other visualization "backends", connecting commands with implementation.  If one backend does not support your desired features, or make the right trade-offs, just switch to another backend with one command.  No need to change your code.  No need to learn something new.  Plots might be the last plotting package you ever learn.
+Plots is a visualization interface and toolset. It sits above other backends, like gr or pyplot, connecting commands with implementation. If one backend does not support your desired features or make the right trade-offs, you can just switch to another backend with one command. No need to change your code. No need to learn a new syntax. Plots might be the last plotting package you ever learn.
```
Comment (daschw) [inline docs/index.md]: maybe replace gr or pyplot with GR or PyPlot

File: docs/index.md [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
 
 My goals with the package are:
 
-- **Powerful**.  Do more with less.  Complex visualizations become easy.
-- **Intuitive**.  Start generating plots without reading volumes of documentation.  Commands should "just work".
-- **Concise**.  Less code means fewer mistakes and more efficient development/analysis.
-- **Flexible**.  Produce your favorite plots from your favorite package, but quicker and simpler.
-- **Consistent**.  Don't commit to one graphics package.  Use the same code and access the strengths of all [backends](/backends).
+- **Powerful**. Do more with less. Complex visualizations become easy.
+- **Intuitive**.   Start generating plots without reading volumes of documentation. Commands should "just work."
```
Comment (daschw) [inline docs/index.md]: Shouldn't it be 'Commands should "just work".' (dot outside the quotes like it was before)?

File: docs/index.md [hunk 3]
```diff
@@ -3,21 +3,21 @@
...
+- **Concise**. Less code means fewer mistakes and more efficient development and analysis.
+- **Flexible**.  . Produce your favorite plots from your favorite package, only quicker and simpler.
```
Comment (daschw) [inline docs/index.md]: There's an extra dot before the sentence here.

File: docs/index.md (hunk 2 above)
Comment (albi3ro) [inline docs/index.md]: American English versus British English.  My grammar checker is set on American conventions.  Though now that I look at it, the British convention looks nicer.

Comment (daschw) [inline docs/index.md]: Ok, I'm not a native english speaker, so I leave this decision to you.

Comment (dpsanders) [inline docs/index.md]: I have never understood the American convention, which seems illogical. 
The rules seem to me to be: (i) "Just work" is a unit that forms part of the sentence; (ii) a sentence ends with a full stop (full point). Thus the full point should be outside the closing quotes. 

I have the feeling that grammarians are perhaps not mathematicians...

Comment (ChrisRackauckas) [inline docs/index.md]: @albi3ro is correct, but @dpsanders is correct that the convention makes no sense.

File: docs/index.md (hunk 3 above)
Comment (ChrisRackauckas) [inline docs/index.md]: I went ahead and handled this.

**Comment by ChrisRackauckas** [comment]
I addressed @daschw 's concerns. Is this ready to merge?

**Comment by daschw** [comment]
Yep, imho it is.

**Comment by ChrisRackauckas** [comment]
Thanks @albi3ro !

**Pull request merged**